pkgs/development/libraries/tracker/default.nix

```diff
@@ -171,7 +171,7 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   meta = with lib; {
-    homepage = "https://wiki.gnome.org/Projects/Tracker";
+    homepage = "https://gitlab.gnome.org/GNOME/tracker";
```

Comment:
https://tracker.gnome.org/

Reply:
Shall I assume the actual homepage is preferred compared to the source code repo? e.g. For https://wiki.gnome.org/Apps/Calculator (TBD in next PR) we use https://apps.gnome.org/Calculator/ instead of https://gitlab.gnome.org/GNOME/gnome-calculator